app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -1,7 +1,10 @@
 <div class="form-actions">
   <% if Sufia.config.analytics %>
-    <%= link_to "Analytics", sufia.stats_work_path(@presenter), id: 'stats', class: 'btn btn-default' %>
-  <% end %>
+    <% if @presenter.model_name.name ==  "GenericWork" %>
```

Comment:
Instead of hardcoding `GenericWork` can you flip the conditional and check for `FileSet` instead?  I want this view to work for other types of works that are not `GenericWorks`.